content/hackathons/2024-05-zagreb/index.mdx

```diff
@@ -89,3 +89,32 @@ As part of the Unikraft community, [Răzvan Deaconescu](https://github.com/razva
 | 12:30-14:00    | Lunch                                                                                                |
 | 14:00-16:00    | Hackathon: Work on Project                                                                           |
 | 16:00-18:00    | Judging and Award Ceremony                                                                           |
+
+### Deploying Cloud Applications using KraftCloud
+
+If you did not create an account already, signup [here](https://console.kraft.cloud/signup) and get a `token`.
+You will be using that in the following sessions.
+
+Once you have a token, follow the steps [here](http://docs.kraft.cloud/quickstart/) to deploy your first unikernel.
+If everything went well, deploy more applications following the links [here](https://docs.google.com/spreadsheets/d/1exnUCaUvDx_FldLOz9HNnGpbmaafMYVL_0Tu6A6AFL4/edit#gid=0).
+
+### Behind the Scenes: Using KraftKit to Operate Unikraft Applications
+
+In the previous session, you deployed application using KraftCloud.
+Now you will build and run some unikernels locally, looking at what `kraft cloud` does behind the scenes.
+
+Follow the steps [here](https://unikraft.org/guides/using-the-app-catalog) and bring the cloud to your machine.
+After you are done with all of them, take a look to some more applications, following the links [here](https://docs.google.com/spreadsheets/d/1exnUCaUvDx_FldLOz9HNnGpbmaafMYVL_0Tu6A6AFL4/edit#gid=89673838).
+
+### Using Docker and Docker-based Filesystems
+
+In order to bring your own application on KraftCloud, you need to build a minimal required filesystem.
+We do that using `docker`, then we will run the application using `kraft`.
```

Comment:
```suggestion
We do that using `docker`.
This is useful both to understand what is happening behind the scenes and to have a test environment for your application.
In case there are issues with KraftCloud / KraftKit, you can use Docker to see if everything is in the right place and to assist in debugging.

```